packages/react-scripts/README.md

```diff
@@ -1,7 +1,92 @@
-# react-scripts
+# backpack-react-scripts
 
-This package includes scripts and configuration used by [Create React App](https://github.com/facebook/create-react-app).<br>
-Please refer to its documentation:
+### **Important:** The currently supported version of **CRA** by `backpack-react-scripts` is up to `v2.1.2`. Versions above this will not work.
 
-- [Getting Started](https://facebook.github.io/create-react-app/docs/getting-started) – How to create a new app.
-- [User Guide](https://facebook.github.io/create-react-app/) – How to develop apps bootstrapped with Create React App.
+This package is a fork of [Create React App](https://github.com/facebookincubator/create-react-app) (specifically the
+`react-scripts` package). It's intended to be used in conjuction with `create-react-app` like so:
+
+```sh
+npx create-react-app my-app --scripts-version=backpack-react-scripts --template @skyscanner/backpack --use-npm
+
+# start your app development like you normally would with `create-react-app`
+cd my-app
+npm start
+```
+
+## What is different in our fork?
+
+- Compilation of uncompiled Backpack components (specifically JSX).
+- Skyscanner specific template with Backpack components integrated out of the box. Published as `@skyscanner/cra-template-backpack`
+- CSS Modules enabled by default for all `.css` & `.scss` files.
+- Ability to create a bundle for server side rending.
+- Automatic chunking is disabled by default.
+- **`css.html` & `js.html`**: New files in the `build/` output folder. These are html partials that include `<script />` and `<link />` references to the various static assets output by webpack. Useful if automatic chunking is turned on and you don't want to worry about order.
+- A bunch of configuration options via `"backpack-react-scripts"` field in `package.json`:
+  - `crossOriginLoading`: Modify the default behaviour, see [docs](https://webpack.js.org/configuration/output/#output-crossoriginloading).
+  - `babelIncludePrefixes`: An array of module name prefixes to opt into babel compilation. Includes `["bpk-", "saddlebag-"]` by default.
+  - `enableAutomaticChunking`: Boolean, opt in to automatic chunking of vendor, common and app code.
+  - `vendorsChunkRegex`: String, Regex for picking what goes into the `vendors` chunk. See `cacheGroups` in webpack docs. Dependent on `enableAutomaticChunking` being enabled
+  - `amdExcludes`: Array of module names to exclude from AMD parsing. Incldues `["lodash"]` by default.
+  - `externals`: exposing the Webpack config to modify externals, see [docs](https://webpack.js.org/configuration/externals/).
+  - `ssrExternals`: Similar to above, but for `ssr.js` only.
+  - `cssModules`: Boolean, true by default.
+
+## Releasing a new version of `backpack-react-scripts`
+
+1. To publish a new version of `backpack-react-scripts`, run the following command:
+
+   ```
+   npm run publish -- --scope backpack-react-scripts
```

Comment:
These instruction are now out of date? (referring to deprecation of `--scope` flag... perhaps they all need to be reviewed.

Reply:
Given for now there is a specifc version where this is still valid no